web/src/stream_popover.ts

```diff
@@ -435,7 +436,7 @@ export async function build_move_topic_to_stream_popover(
         );
     }
 
-    function update_submit_button_disabled_state(select_stream_id: number): void {
+    function update_submit_button_disabled_state(select_stream_id?: number): void {
```

Comment:
Why is this parameter made optional?

Reply:
I introduced it as if we know that only the first block will be implemented, then there is no need for select_stream_id.

Reply:
We need that only when there is some case where the function is called without passing the parameter, which I don't think is what we do currently.